admin/class-admin-apple-meta-boxes.php

```diff
@@ -138,6 +138,16 @@ public static function save_post_meta( $post_id ) {
 		}
 		update_post_meta( $post_id, 'apple_news_is_sponsored', $is_sponsored );
 
+		if ( ! empty( $_POST['apple_news_maturity_rating'] ) ) {
+			$maturity_rating = sanitize_text_field( $_POST['apple_news_maturity_rating'] );
+			if ( ! in_array( $maturity_rating, self::$maturity_ratings ) ) {
+				$maturity_rating = '';
+			}
+		}
+		if ( ! empty( $maturity_rating ) ) {
```

Comment:
why not move this within the above if statement so it avoids this empty check if the _POST item is empty?